internal/cobraprompt/prompt.go

```diff
@@ -49,9 +51,39 @@ type CobraPrompt struct {
 	OnErrorFunc func(err error)
 }
 
+var ExitError = errors.New("exit prompt")
+
+// Terminal breaks on os.Exit for go-prompt https://github.com/c-bata/go-prompt/issues/59#issuecomment-376002177
+func exitPromptSafely() {
+	panic(ExitError)
+}
+
+func handleExit() {
+	switch v := recover().(type) {
+	case nil:
+		return
+	case error:
+		if v == ExitError {
```

Comment:
Let's use `if errors.Is(v, ErrExit)` instead.

Reply:
Thanks for the suggestion, https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/bdd565a3b18e25cbb9e967db21a395523885a7e1